desktop/package.json

```diff
@@ -86,5 +88,8 @@
     "webpack-dev-server": "1.14.1",
     "webpack-hot-middleware": "2.10.0",
     "webpack-target-electron-renderer": "0.4.0"
+  },
+  "optionalDependencies": {
+    "fsevents": "1.0.11"
```

Comment:
Yes it fails:

```
npm ERR! node v4.1.2
npm ERR! npm  v2.14.4
npm ERR! code EBADPLATFORM
npm ERR! notsup Unsupported
npm ERR! notsup Not compatible with your operating system or architecture: fsevents@1.0.11
npm ERR! notsup Valid OS:    darwin
npm ERR! notsup Valid Arch:  any
npm ERR! notsup Actual OS:   linux
npm ERR! notsup Actual Arch: x64
```

Now I remember the reason behind this: the shrinkwrap file specifies that fsevents is installed. We need to override that by specifying the dep as optional at the top level.